src/main/java/seedu/address/model/Model.java

```diff
@@ -95,6 +97,16 @@ public interface Model {
 
     /////////////////////////
 
+    /** Returns an unmodifiable view of the filtered question list */
```

Comment:
This too 😄 "question list" to "student list"

Reply:
👍